### ISSUE TITLE
naive-ui > seemly > @types/jest breaks typescript usage with mocha

### ISSUE THREAD

**Issue by undefined-moe** [TuSimple/naive-ui#3267]
### TuSimple/naive-ui version (版本)
2.31.0

### Vue version (Vue 版本)
3.2.37

### Browser and its version (浏览器及其版本)
any

### System and its version (系统及其版本)
typescript@4.6.4

### Node version (Node 版本)
node v14.19.0

### Reappearance link (重现链接)
https://codesandbox.io/s/not-required

### Reappearance steps (重现步骤)
build a typescript project with both `naive-ui` and `@types/mocha` installed

### Expected results (期望的结果)
build successfully

### Actual results (实际的结果)
```
node_modules/@types/mocha/index.d.ts:2725:13 - error TS2403: Subsequent variable declarations must have the same type.  Variable 'test' must be of type 'It', but here has type 'TestFunction'.

2725 declare var test: Mocha.TestFunction;
                 ~~~~

  node_modules/@types/jest/index.d.ts:43:13
    43 declare var test: jest.It;
                   ~~~~
    'test' was also declared here.

...lots of similar errors omitted...
```

**Comment by 07akioni** [comment]
reproduction needed.

**Comment by undefined-moe** [comment]
I don't understand why a minimal reproduction is needed as its a problem related to dep tree, not the code.

Buy anyway, here it is: https://github.com/undefined-moe/test

clone the repo, run `yarn && yarn reproduce` (or npm, not sure if this example works with plug-n-play)

**Comment by undefined-moe** [comment]
This issue is caused by package [seemly](https://www.npmjs.com/package/seemly?activeTab=dependencies) in your dependencies, which included (by accident) `@types/jest` to their deps.

Possiable solutions are:

- Notify the upstream to publish a patch version
- Switch to another library

**Comment by 07akioni** [comment]
> This issue is caused by package [seemly](https://www.npmjs.com/package/seemly?activeTab=dependencies) in your dependencies, which included (by accident) `@types/jest` to their deps.
> 
> Possiable solutions are:
> 
> * Notify the upstream to publish a patch version
> * Switch to another library

You are correct

**Comment by 07akioni** [comment]
> This issue is caused by package [seemly](https://www.npmjs.com/package/seemly?activeTab=dependencies) in your dependencies, which included (by accident) `@types/jest` to their deps.
> 
> Possiable solutions are:
> 
> * Notify the upstream to publish a patch version
> * Switch to another library

I've released `seemly@0.3.5`. I think you can reinstall naive-ui (remove lock file first) to solve your problem.

**Comment by undefined-moe** [comment]
works for me :)
